iks/provider/provider_test.go

```diff
@@ -242,7 +239,8 @@ func TestOpenSession(t *testing.T) {
 
 	vpcp, err := GetTestProvider(t, logger)
 	assert.Nil(t, err)
-	sessn, err := vpcp.OpenSession(context.Background(), provider.ContextCredentials{
+	//nolint:gosec
+	vpcp.OpenSession(context.Background(), provider.ContextCredentials{
```

Comment:
how to fix security fix?

Reply:
vpcp.OpenSession returns err. We are not checking if err is nil or not.
I added 
require.NoError(t, err)
assert.NotNil(t, sessn)
which will check err should be nil.

Reply:
Done.